stable/grafana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 1.12.0
+version: 1.12.1
```

Comment:
If I'm not mistaken, when performing an update, the 'old' PVC will not be (re-)used and the StatefulSet controller will create a new one. En plus, the old PVC will not be removed by Tiller. As such, would this warrant a major bump?

Reply:
Yeah all true. Actually this got me thinking a bit more - i guess using StatefulSet won't really work with scaling up grafana to multiple replicas (as each pod will have it's own storage, inconsistencies will start happening - e.g. one pod will store user-created dashboard, while others wont have it). Current behavior with deployment + persistence enabled is that all pods connect to the same volume (which is not optimal as well), but at least scaling fails if using a volume not supporting multiple writes.

Closing this one for now.